paper/paper.bib

```diff
@@ -59,49 +55,39 @@ @Inbook{Koranne2011
 	doi={10.1007/978-1-4419-7719-9_6},
 	url={https://doi.org/10.1007/978-1-4419-7719-9_6},}
 
-@techreport{Sola18,
-	Address = {Barcelona},
-	Archiveprefix = {arXiv},
-	Arxivid = {1812.01537},
-	Author = {Sol{\`{a}}, Joan and Deray, Jeremie and Atchuthan, Dinesh},
-	Eprint = {1812.01537},
-	Institution = {Institut de Rob{\`{o}}tica i Inform{\`{a}}tica Industrial},
-	Isbn = {1812.01537v4},
-	Number = {IRI-TR-18-01},
-	Title = {{A micro Lie theory for state estimation in robotics}},
-	Url = {https://arxiv.org/abs/1812.01537v4},
-	Year = {2018},
-	Bdsk-Url-1 = {https://arxiv.org/abs/1812.01537v4}}
+@article{Sola18,
+    author = {Sol{\`{a}}, Joan and Deray, Jeremie and Atchuthan, Dinesh},
+    title = {{A micro Lie theory for state estimation in robotics}},
+    archivePrefix = {arXiv},
+    eprint = {1812.01537},
+    primaryClass = {cs.RO},
+    year = 2018,
+    url = {https://arxiv.org/abs/1812.01537v4},
+}
 
 @techreport{EADE-Lie,
 	Author = {Ethan Eade},
-	Date-Added = {2018-08-03 15:46:41 +0000},
-	Date-Modified = {2018-08-03 15:47:03 +0000},
 	Title = {Lie Groups for {2D} and {3D} Transformations},
-  journal = {URL http://ethaneade. com/lie. pdf, revised Dec},
+  url = {http://ethaneade.com/lie.pdf},
   year = {2013},
   institution = {Cambridge University},}
 
 @book{CHIRIKJIAN-11,
 	Author = {Chirikjian, G.S.},
-	Date-Added = {2019-01-31 12:40:38 +0100},
-	Date-Modified = {2019-01-31 12:41:30 +0100},
 	Isbn = {9780817649449},
 	Lccn = {2009933211},
 	Publisher = {Birkh{\"a}user Boston},
 	Series = {Applied and Numerical Harmonic Analysis},
 	Title = {Stochastic Models, Information Theory, and Lie Groups, Volume 2: Analytic Methods and Modern Applications},
 	Url = {https://books.google.ch/books?id=hZ1CAAAAQBAJ},
 	Year = {2011},
-	Bdsk-Url-1 = {https://books.google.ch/books?id=hZ1CAAAAQBAJ},}
+}
 
 @book{BARFOOT-17-Estimation,
 	Author = {Barfoot, Timothy D.},
```

Comment:
`Surname, Name` instead of `Name, Surname`

Reply:
the two allowed styles are ```surname, name``` or ```name surname```  there's no need to be consistent; both will be interpreted correctly

Reply:
OK